ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -154,6 +151,22 @@ public void shouldFailCreateExistingTopic() {
     verify(adminClient);
   }
 
+  @Test
+  public void shouldFailCreateTopicNoAclsSet() {
+    expectedException.expect(TopicAuthorizationException.class);
+    expectedException.expectMessage(
+            AclErrorMessageUtil.generateErrorMessage(AclOperation.CREATE, topicName1)
+    );
+
+    expect(adminClient.listTopics()).andReturn(getEmptyListTopicResult());
+    expect(adminClient.createTopics(anyObject())).andReturn(createTopicAuthorizationException(AclOperation.CREATE));
```

Comment:
It doesn't matter what the error message of the `TopicAuthorizationException` returned by the `adminClient` is, right? If so, we can clean up this test by making it return a generic `TopicAuthorizationException` rather than worrying about returning a specific error message with it. In fact, that's better since then we verify that `KafkaTopicClientImpl` is populating the correct error message, rather than simply passing the inner error message along.

Same comment for the describe topic test below.